### PR TITLE
Improve explanation of image formatting in `Article styling criteria/Formatting`

### PR DESCRIPTION
**Pull request by Samaoo**
- edit examples for `Images`
- add explanation for text between square brackets and quotes

### CODE REVIEW

File: wiki/Article_styling_criteria/Formatting/en.md
Comment (cl8n) [inline wiki/Article_styling_criteria/Formatting/en.md:616]: flags are kinda bad examples at this point because they're supporting icons that don't need alt text.

I think a good example would be the placement crowns from tournaments, since they showcase the difference between alt & title text

File: wiki/Article_styling_criteria/Formatting/en.md
Comment (cl8n) [inline wiki/Article_styling_criteria/Formatting/en.md:631]:
```suggestion
The text in the first pair of square brackets (*[alternative text](#alt-text)*) is optional and should describe the image literally. It is used by screen readers or when the image fails to load.

The text in the quotation marks (*title text*) is optional and should give additional context to the image or indicate its meaning. It is displayed when hovering over the image and set as the [caption](#captions) if applicable.
```

Comment (cl8n) [inline wiki/Article_styling_criteria/Formatting/en.md:631]: not sure if it matters but this feels a little repetitive with the `Alt text` and `Captions` sections. maybe this should stick to explaining the formatting only, and suggested usage goes in the other sections? maybe combine all the sections into this one? maybe it's fine as-is?

I'll just wait for another review...

**Review by cl8n — CHANGES_REQUESTED** [review]

**Comment by cl8n** [comment]
also try to use commit names that describe what you're changing, rather than just the file name.

File: wiki/Article_styling_criteria/Formatting/en.md
Comment (TicClick) [reply wiki/Article_styling_criteria/Formatting/en.md:631]: imo merging all three sections into a single one makes more sense. it would also allow for a shorter explanation, such as:

```markdown
Images are blah blah and have the following format:

![Alt text](/wiki/path/to/image.png "Title text")

Where:

- `Alt text` is ... . It is optional and does ... and ... .
- `Title text` is ... . It is optional and does ... and ... . It will be rendered as the image's caption if ... and ... .
```

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Article_styling_criteria/Formatting/en.md
Comment (TicClick) [reply wiki/Article_styling_criteria/Formatting/en.md:616]: +1

**Comment by cl8n** [comment]
will update this one myself, noticing other things in this section I want to write better

File: wiki/Article_styling_criteria/Formatting/en.md
Comment (cl8n) [reply wiki/Article_styling_criteria/Formatting/en.md]: didn't do exactly this but I think I made it read well from top to bottom now

**Review by TicClick — APPROVED** [review]

**Comment by TicClick** [comment]
that's better, thank you both. @cl8n I am fine with the changes, wanna merge?

**Review by TPGPL — DISMISSED** [review]
lgtm

**Review by cl8n — APPROVED** [review]

**Pull request merged**
